util/generate_compilation_db.py

```diff
@@ -103,10 +104,16 @@ def main(args):
         '--output=jsonproto',
         args.target,
     ]
-    completed_process = subprocess.run(bazel_aquery_command,
-                                       stdout=subprocess.PIPE,
-                                       stderr=subprocess.PIPE,
-                                       check=True)
+    try:
+        completed_process = subprocess.run(bazel_aquery_command,
+                                           stdout=subprocess.PIPE,
+                                           stderr=subprocess.PIPE,
+                                           check=True)
+    except subprocess.CalledProcessError as e:
+        print(e.stderr.decode('utf-8'), file=sys.stderr)
+        raise
+    except BaseException:
+        raise
```

Comment:
LGTM

Out of curiosity, what does this accomplish?

Reply:
You could technically drop the `except BaseException: raise` part without changing the program's behavior. But if we deleted it, that would imply to the reader that `subprocess.CalledProcessError` is the only kind of exception that can be thrown by `subprocess.run`.